tests/lib/bin/session-tool

```diff
@@ -36,6 +55,68 @@ while [ $# -gt 0 ]; do
 	esac
 done
 
+case "$action" in
+	prepare)
+		# Make /var/lib/systemd writable so that we can get linger enabled.
+		# This only applies to Ubuntu Core 16 where individual directories were
+		# writable. In Core 18 and beyod all of /var/lib/systemd is writable.
+		case "$SPREAD_SYSTEM" in
+			ubuntu-core-16-*)
+				mount tmpfs -t tmpfs /var/lib/systemd/
```

Comment:
a comment that this setup is creating a writable mimic for /var/lib/systemd (with original files from rfkill and random-seed) would be helpful to the reader